src/data/ab-attrs/post-knock-out-stat-stage-change-ab-attr.ts

```diff
@@ -15,13 +15,7 @@ export class PostKnockOutStatStageChangeAbAttr extends PostKnockOutAbAttr {
     this.stages = stages;
   }
 
-  override applyPostKnockOut(
-    pokemon: Pokemon,
-    _passive: boolean,
-    simulated: boolean,
-    _knockedOut: Pokemon,
-    _args: any[],
-  ): boolean {
+  override applyPostKnockOut(pokemon: Pokemon, _passive: boolean, simulated: boolean, _knockedOut: Pokemon): boolean {
```

Comment:
NIT:
```suggestion
  override applyPostKnockOut(pokemon: Pokemon, _passive: boolean, simulated: boolean, _knockedOutPokemon: Pokemon): boolean {
```

The name is bugging me. If I were to see that variable name I'd expected a boolean or so